library/src/main/java/org/xmtp/android/library/Group.kt

```diff
@@ -184,14 +185,30 @@ class Group(val client: Client, private val libXMTPGroup: FfiGroup) {
         }
     }
 
-    fun memberAddresses(): List<String> {
-        return libXMTPGroup.listMembers().map { it.accountAddress }
+    suspend fun addMembersByInboxId(inboxIds: List<String>) {
+        try {
+            libXMTPGroup.addMembersByInboxId(inboxIds)
+        } catch (e: Exception) {
+            throw XMTPException("User does not have permissions", e)
```

Comment:
There are other things that can go wrong here besides permissions checks. For example, invalid inbox_ids. Or the network being down. We should probably make the error message more generic and rely on the exception from libxmtp

Reply:
I think theres gonna be a big pass with permissions when @cameronvoell brings his admin permissions work over. But I'll make this more generic in the meantime.